src/Console/Command/FixCommand.php

```diff
@@ -506,6 +434,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param string          $format
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return ReportInterface
+     */
+    protected function createReport($format, InputInterface $input, OutputInterface $output)
+    {
+        $reportClass = self::$reports[$format];
+
+        /** @var ReportInterface $report */
+        $report = new $reportClass();
+        $report->setDecoratedOutput($output->isDecorated());
```

Comment:
I'm fine for having one class with option, like mentioned here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1824#discussion_r56498208